src/content/docs/tutorial-service-level-mgmt/establish-output-sli.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: "Establish an output performance SLI"
+metaDescription: "New Relic observability maturity series: our service level management guide shows you how to easily measure and communicate the overall health, performance, and quality of your digital products and services to all stakeholders."
+redirects:
+freshnessValidatedDate: never
+---
+
+import apmOverviewScreen from 'images/apm_screenshot-full_overview-screen.webp'
+
+import apmServiceBoundary from 'images/apm_screenshot-crop_service-boundary.webp'
+
+import apmLatencyBaseline from 'images/apm_screenshot-full_latency-baseline.webp'
+
+import apmSLStartFromApm from 'images/apm_screenshot-full_sl-start-from-apm.webp'
+
+The ability to manage and query vast amounts of data efficiently is one of the core capabilities of New Relic. We measure query output health the same as you might measure your client-facing capabilities: by how fast and error-free HTTP query requests receive responses. We refer to this as output performance, and you can measure use SLIs to track this. Once you set up an output performance SLI using the procedure below, you can track both response times and the error rate of your applications.
+
+<Steps>
+    <Step>
+## Identify your service
+
+Make sure that you've done the following:
+* Instrumented your primary applications with [New Relic APM agents](/docs/apm/new-relic-apm/getting-started/introduction-apm).
+* Named your applications with a familiar naming convention, as outlined in our [Service characterization guide](/docs/new-relic-solutions/observability-maturity/operational-efficiency/sc-implementation-guide/#improvement-process).
+* Located your application in our [entity explorer](/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts#find).
```

Comment:
I wonder if we should reframe this. Maybe these bullets should be a "If you can't find your app in the entity explorer, make sure you've done the following, then come back to this tutorial." 

tbh being sent to something with a 'decision matrix' in step 1 is pretty intimidating!